__init__.py

```diff
@@ -129,8 +131,28 @@ def _b64(b):
         domain_key_file.write(domain_key_contents)
 
       log.info("Generate csr...")
-      proc = subprocess.Popen(["openssl", "req", "-new", "-sha256", "-key", domain_key, "-subj", "/CN=" + domain],
-                              stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+      if len(domains) > 1:
+        domain_config_file = tempfile.NamedTemporaryFile()
+        default_conf = open("/etc/ssl/openssl.cnf").readlines()
```

Comment:
Is there a way you can make this platform independent? /etc/sll/openssl.cnf only works on Linux ...